File: C++/VI to IX - Programming Exercise (Lottery Number Generator).cpp
#include <iostream>
#include <cstdlib>
#include <ctime>

using namespace std;

int main()
{
    string name;
    srand(time(0));
    int lottery_number[7];
    for(int i = 0; i < 7; i++)
    {
        lottery_number[i] = rand() % 10;
    }
    cout << "Full Name: ";
    getline(cin, name);
    
    cout << "Generated Lottery Numbers: ";
    for(int i = 0; i < 7; i++)
    {
        cout << lottery_number[i] << " ";
    }
    cout << endl;
    return 0;
}

File: Java/VI to IX - Programming Exercise (Lottery Number Generator).java
import java.util.Scanner;
import java.util.Random;

public class Main {
    public static void main(String[] args) {
        Random rand = new Random();
        int[] lotteryNumbers = new int[7];
        for (int i = 0; i < 7; i++) {
            int randomNum = rand.nextInt(10);
            lotteryNumbers[i] = randomNum;
        }
        
        Scanner scan = new Scanner(System.in);
        System.out.println("Full Name: ");
        String name = scan.nextLine();
        
        System.out.print("Generated Lottery Numbers: ");
        for (int num : lotteryNumbers) 
        {
            System.out.print(num + " ");
        }
    }
}
